components/logins/android/src/main/java/mozilla/appservices/logins/DatabaseLoginsStorage.kt

```diff
@@ -218,6 +272,44 @@ class DatabaseLoginsStorage(private val dbPath: String) : AutoCloseable, LoginsS
         }
     }
 
+    private fun gatherSnapshotMetrics() {
+        // XXX TODO: ask the rust code to return snapshot metrics from the db.
+        LoginsStoreMetrics.loginLastUsedDays.accumulateSamples(longArrayOf(1L,2L,3L,4L,5L))
```

Comment:
Please note that this API is reserved for GeckoView integration and should not be used.